### ISSUE TITLE
Some Key Bindings don't work

### ISSUE THREAD

**Issue by moosichu** [slynch8/10x#1708]
- I’d like to be able to bind Control Alt - to a command, however this combination doesn't seem to work. If I enable "Log Keys" it doesn't show up either for reference.
- Similarly, I'm having the same issue with Control Break/Pause, with the Break/Pause key not showing with "Log Keys" enabled (with Control held or not).

Similarly, there might be other key bindings/combinations that don't work, but these are the only ones that I use that haven't for me (so far)!

**Comment by slynch8** [comment]
No, nothing changed. Not sure why it started working, but glad it did :-)
